upload/library/Sidane/Threadmarks/Model/Threadmarks.php

```diff
@@ -3,6 +3,102 @@
 class Sidane_Threadmarks_Model_Threadmarks extends XenForo_Model
 {
 
+  public function getMenuLimit(array $thread, array $nodePermissions = null, array $viewingUser = null)
+  {
+    $this->standardizeViewingUserReferenceForNode($thread['node_id'], $viewingUser, $nodePermissions);
+    
+    $menulimit = XenForo_Permission::hasContentPermission($nodePermissions, 'permission_forum_sidane_tm_menu_limit'); 
```

Comment:
This is intentional, as it is a content aware permission check. So you have different menu limits per node, not just per user group.

You need to set the "Menu limit" on the user group or the forum. I'm going to have to update the installer to set the a default value on the Registered group (or unregistered group) to a sane value. 

As zero causes it to be cut of, and doesn't make this addon "just work"